fastlane/Fastfile

```diff
@@ -49,7 +50,9 @@ before_all do |lane|
   unless lane == :test_without_building
     # Check that the env files exist
     unless is_ci || File.file?(USER_ENV_FILE_PATH)
-      UI.user_error!("~/.wpios-env.default not found: Please copy fastlane/env/user.env-example to #{USER_ENV_FILE_PATH} and fill in the values")
+      example_path = 'fastlane/env/user.env-example '
+      msg = "#{ENV_FILE_NAME} not found: Please copy #{example_path} to #{USER_ENV_FILE_PATH} and fill in the values"
+      UI.user_error!(msg)
```

Comment:
I did this before disabling the line length violation. Still, it DRYes a bit (see `ENV_FILE_NAME`) and I don't think it hurst the code, so I decide to leave it.

Reply:
I'm not sure we need to disable the line length violation – splitting method calls onto multiple lines does improve readability anyway 🤔 

WDYT?

Reply:
Yeah true. I mainly disabled it because of long string lines like https://github.com/wordpress-mobile/wordpress-ios/blob/fastlane-rubocop/fastlane/Fastfile#L419.

There's probably a Ruby way to split the string without messing up the horizontal line length 🤔